baybe/surrogates/gaussian_process/__init__.py

```diff
@@ -0,0 +1,7 @@
+"""Gaussian process surrogates."""
+
+from baybe.surrogates.gaussian_process.core import GaussianProcessSurrogate
```

Comment:
Will there ever be an actual other class for Gaussian Process Surrogates? What is the plan here?

Reply:
Perhaps, this might at some point split up into subclasses like singletask, multitask, etc ... But nothing planned. Why are you asking? FYI: the fact that there is only one element contained so far is not a problem – the purpose is still the same: to define what is contained in the package namespace